README.md

```diff
@@ -48,27 +48,27 @@ kubectl apply -f ./update-operator.yaml
 Once the deployment's resources are in place, there is one more step needed to schedule and place the required pods on Bottlerocket nodes.
 By default - in the suggested deployment, each Workload resource constrains scheduling of the update operator by limiting pods to Bottlerocket nodes based on their labels.
 These labels are not applied on nodes automatically and will need to be set on each using `kubectl`.
-Deployments match on `bottlerocket.aws/platform-version` (a host compatibility label) to determine which pod to schedule.
+Deployments match on `bottlerocket.aws/updater-interface-version` (a host compatibility label) to determine which pod to schedule.
```

Comment:
Let's update the wording a bit to briefly explain why the new name is what it is :+1: 

Maybe something like this?

```suggestion
The agent relies on each node's updater components and schedules its pods based on their interface supported.
The node indicates its updater interface version in a label called `bottlerocket.aws/updater-interface-version`.
Agent deployments, respective to the interface version, are scheduled using this label and target only a single version in each.
```